depfu/bot/lib/depfu.rb

```diff
@@ -82,8 +82,13 @@ def build_comment(pull_request)
           msg << "- Package #{gem.name} is already up to date in d:l:r:e (#{gem.version}), update link reference with `osc setlinkrev OBS:Server:Unstable rubygem-#{gem.name}`.\n"
         elsif factory_auto_packages.any? { |package| package.name.end_with?(gem.name) && package.version == gem.version }
           submit_request = factory_auto_packages.find { |package| package.name.end_with?(gem.name) && package.version == gem.version }.submit_requests.first
-          msg << "- There is already a submit request for package #{gem.name} (#{gem.version}). "
-          msg << "Please review and accept [#{submit_request.number}](https://build.opensuse.org/request/show/#{submit_request.number}).\n"
+          if submit_request
+            msg << "- There is already a submit request for package #{gem.name} (#{gem.version}). "
+            msg << "Please review and accept [#{submit_request.number}](https://build.opensuse.org/request/show/#{submit_request.number}).\n"
+          else
+            msg << "- There is already a package in https://build.opensuse.org/user/show/factory-auto project but no submit request. "
```

Comment:
https://build.opensuse.org/user/show/factory-auto is the page of the factory-auto user. Couldn't it be better to link to link to https://build.opensuse.org/project/show/home:factory-auto:branches:devel:languages:ruby:extensions , instead?

Reply:
Good point, updated